stable/redmine/templates/_helpers.tpl

```diff
@@ -196,3 +196,53 @@ Usage:
         {{- tpl (.value | toYaml) .context }}
     {{- end }}
 {{- end -}}
+
+{{/*
+Create a default fully qualified mail receiver app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "redmine.mailReceiver.fullname" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s-mail-receiver" .Release.Name $name | trunc 40 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the proper Redmine mail Receiver image name
+*/}}
+{{- define "redmine.mailReceiver.image" -}}
+{{- $registryName := .Values.mailReceiver.image.registry -}}
+{{- $repositoryName := .Values.mailReceiver.image.repository -}}
+{{- $tag := .Values.mailReceiver.image.tag | toString -}}
+{{/*
+Helm 2.11 supports the assignment of a value to a variable defined in a different scope,
+but Helm 2.9 and 2.10 doesn't support it, so we need to implement this if-else logic.
+Also, we can't use a single if because lazy evaluation is not an option
+*/}}
+{{- if .Values.global }}
+    {{- if .Values.global.imageRegistry }}
+        {{- printf "%s/%s:%s" .Values.global.imageRegistry $repositoryName $tag -}}
+    {{- else -}}
+        {{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+    {{- end -}}
+{{- else -}}
+    {{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Mail Receiver CronJob labels
+*/}}
+{{- define "redmine.mailReceiver.labels" -}}
+app.kubernetes.io/name: {{ template "redmine.mailReceiver.fullname" . }}
```

Comment:
This label should use `{{ template "redmine.mailReceiver.name" . }}` instead of `{{ template "redmine.mailReceiver.fullname" . }}`

Reply:
Thank you. Fixed